tests/flow/test_ts_mrange.py

```diff
@@ -25,6 +25,19 @@ def test_mrange_with_expire_cmd():
             assert(len(reply)>=0 and len(reply)<=3)
         assert r.execute_command("PING")
 
+def testWithMultiExec(env):
+    env = Env()
+    if(not env.isCluster):
+        env.skip()
+    with env.getConnection() as r:
+        r.execute_command('multi', )
+        r.execute_command('TS.mrange', '-', '+', 'FILTER', 'name=bob')
+        try:
+            r.execute_command('exec')
+            env.assertTrue(True, message='Did not get error when running mrange in multi exec')
```

Comment:
@OfirMos you should use pytest's assert raises
https://docs.pytest.org/en/6.2.x/reference.html#pytest.raises